singularity-eos/eos/eos_spiner_rho_sie.hpp

```diff
@@ -88,6 +136,8 @@ class SpinerEOSDependsRhoSie : public EosBase<SpinerEOSDependsRhoSie> {
   static constexpr int NGRIDS = 3;
   using Grid_t = Spiner::PiecewiseGrid1D<Real, NGRIDS>;
   using DataBox = Spiner::DataBox<Real, Grid_t>;
+  using TransformData = Transform_Data;
+  using Transformer = TransformerT<TransformData>;
```

Comment:
with my suggestion above, this would become
```suggestion
  using TransformData = TransformDataContainer<DataBox>;
  using Transformer = TransformerT<TransformDataT>;
```